operator/cmd/mesh/manifest-apply.go

```diff
@@ -139,6 +140,21 @@ func InstallManifests(setOverlay []string, inFilenames []string, force bool, dry
 	if err != nil {
 		return err
 	}
+
+	serverVersion, err := clientset.Discovery().ServerVersion()
+	if err != nil {
+		return fmt.Errorf("error getting k8s version: %w", err)
+	}
+	ok, err := k8sversion.CheckKubernetesVersion(serverVersion)
+	if err != nil {
+		return fmt.Errorf("error comparing k8s version: %w", err)
+	}
+	if !ok {
+		return fmt.Errorf(
+			"the kubernetes version is not supported by this version of Istio. The minimum supported kubernetes version is %s",
```

Comment:
I think the following message will make more sense?
```
the Kubernetes version 1.15 is not supported by Istio 1.7.  The minimum supported Kubernetes version is 1.17.
```

Reply:
Agreed.

Reply:
@shamsher31 done.